src/ring_theory/subsemiring.lean

```diff
@@ -339,6 +340,66 @@ lemma closure_eq_of_le {s : set R} {t : subsemiring R} (h₁ : s ⊆ t) (h₂ :
   closure s = t :=
 le_antisymm (closure_le.2 h₁) h₂
 
+end subsemiring
+
+namespace submonoid
+
+/-- The additive closure of a submonoid is a subsemiring. -/
+def to_subsemiring (M : submonoid R) : subsemiring R :=
```

Comment:
`to_subsemiring` sounds like a projection to me (i.e., something that might underlie a coercion).
Should this be `submonoid.subsemiring_closure`?

Reply:
I mentioned this above, where I think I suggested `add_closure` or `additive_closure`. I agree, `to_subsemiring` sounds like it just moves some proofs around without actually changing the underlying data.

Reply:
Oh, I had missed the comment above and I agree: in fact, I think that I understood better a convention, after I changed the name of this lemma!

Thank you for your comments!